packages/core/types/src/schema/index.ts

```diff
@@ -122,12 +178,29 @@ export type AttributeNamesWithTarget<TSchemaUID extends UID.Schema> = Extract<
   AttributeNames<TSchemaUID>
 >;
 
+/**
+ * Extracts the names of all required attributes from a given schema.
+ *
+ * @remark `
+ RequiredAttributeNames` screens attributes based on the {@link Attribute.Required} property,
```

Comment:
```suggestion
 * @remark `RequiredAttributeNames` screens attributes based on the {@link Attribute.Required} property,
```
Same comment everywhere. Not sure how this will be parsed across multiple lines, especially since we're probably generating docs from these